docs/_includes/repl.html

```diff
@@ -0,0 +1,53 @@
+<form>
+    <div>
+        <label>Script:</label>
+        <br>
+        <textarea rows="10" cols="70" id="script">print("Hello World!")</textarea>
+        <br><br>
+        <button onclick="clearInput(); return false;">
+            Clear Input
+        </button>
+        <button onclick="executeScript(); return false;">
+            Run
+        </button>
+    </div>
+    <br><br>
+    <div>
+        <label>Output:</label>
+        <br>
+        <textarea readonly rows="10" cols="70" id="output"></textarea>
+        <br><br>
+        <button onclick="clearOutput(); return false;">
+            Clear Output
+        </button>
+    </div>
+</form>
+
+<script>
+    function output(text) {
+        document.getElementById("output").value += new Date().toLocaleTimeString() + ": " + text + "\n";
+    }
+
+    var Module = {
+        'print': function (msg) { output(msg) },
+        'printErr': function (err) { } // potentially unused?
+    };
+
+    function clearInput() {
+        document.getElementById("script").value = "";
+    }
+
+    function clearOutput() {
+        document.getElementById("output").value = "";
+    }
+
+    function executeScript() {
+        try {
+            Module.ccall('executeScript', null, ['string'], [document.getElementById("script").value]);
+        }
+        catch (e) {
+            output(Module.ccall('getExceptionFromPtr', 'string', ['number'], [e]).replace('stdin', '[ERROR]'));
+        }
+    }
+</script>
+<script async src="assets/luau/Luau.Repl.Web.js"></script>
```

Comment:
Thinking through the update flow. I'm curious, is there a way to get this via artifact store? Ideally our `release` build action would publish this as an artifact so that we never have to update anything ourselves.

Reply:
I thought about this. I'm not too sure on what the best way to do this is yet. I was hoping to get an opinion from you (and/or) the community. The idea you've put forward seems feasible. I'll give it a try! :)

Reply:
I don't actually think this is possible. The best way to do this is to have some script which builds and runs the site. While it is being built, it should fetch the latest release of luau.wasm and luau.js from here and put them in the assets dir. 

Alternatively, we can follow the method I've been using so far: build Luau.Repl.CLI using emscripten which copies the files over to docs, and then serve as normal with jekyll. 

What do you think?

Reply:
So this problem clearly will need a bit more research...

How about this: let's remove the build action (it doesn't build for us right now due to security restrictions on actions allowed, we'd need to see if we can lift that or just use emsdk directly) and remove the link to the new demo page from all navigation while keeping the demo page itself.

This would let us merge this PR without having any adverse effects, as even though the .js file isn't published anywhere, the page that links to it won't be accessible. We can then look into what a good build / publish process is on our end.

Reply:
Sure, no problem! I'll do that now.